runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2853,6 +2853,9 @@ gcInitializeDefaults(J9JavaVM* vm)
 	if (gc_policy_gencon == extensions->configurationOptions._gcPolicy) {
 		/* after we parsed cmd line options, check if we can obey the request to run CS (valid for Gencon only) */
 		if (extensions->concurrentScavengerForced) {
+#if defined(J9VM_ARCH_X86) && defined(J9VM_ENV_DATA64) && defined(J9VM_GC_COMPRESSED_POINTERS)
+            extensions->concurrentScavenger = true;
```

Comment:
Please use tabs here rather than spaces so the indentation of this is correct.

Reply:
Good catch. Updated.